Dockerfile

```diff
@@ -1,3 +1,18 @@
+FROM node:6 as frontend
+
+# these build args are turned into env vars
+# and used in bin/build_frontend.sh
+ARG FRONTEND_SENTRY_PUBLIC_DSN=UNSET_DSN
+ENV FRONTEND_SENTRY_PUBLIC_DSN=${FRONTEND_SENTRY_PUBLIC_DSN}
+ARG CI=false
+ENV CI=${CI}
```

Comment:
This repetition is gross but I guess necessary evil. Well done for figuring it out!

Reply:
unfortunately, I believe this is indeed a necessary evil.